Makefile

```diff
@@ -28,11 +26,10 @@ wasm:
 		-s MALLOC="emmalloc" \
 		-s INCOMING_MODULE_JS_API=['instantiateWasm','locateFile']\
 		-s EXPORT_NAME="yoga" \
-		-s "DEFAULT_LIBRARY_FUNCS_TO_INCLUDE=['memcpy','memset','malloc','free','strlen']" \
```

Comment:
seems like this option comes from `asm.js` compilation

it doesn't work with the closure compiler without `EXPORTED_FUNCTIONS` param so I removed it.

docs: https://github.com/emscripten-core/emscripten/blob/main/src/settings.js#L995-L1004